docker-compose.yml

```diff
@@ -0,0 +1,134 @@
+version: '3'
+
+services:
+    data-hub-dags:
+        environment:
+            GOOGLE_APPLICATION_CREDENTIALS: /tmp/credentials.json
+        volumes:
+            - ./credentials.json:/tmp/credentials.json
```

Comment:
this is probably the same in other dags repo. But it might be worth moving the CI specific mounts to a `docker-compose.ci.yml`?

Reply:
`CI` is the default, and `dev` is the specialized with the https://github.com/elifesciences/data-hub-ejp-xml-pipeline/blob/init/docker-compose.dev.override.yml, and in my opinion, these are the two deployment environment which are important here

Reply:
I would argue that `CI` is a special case like `dev` is. The default should be for deployment and running. I find it a bit more difficult to follow because the special cases are meant to be mostly additive. But in this case it's replacing a mount. But it's not a big issue, perhaps just my opinion.

Reply:
not a lot of work, need to modify the makefile as well , done that that already

Reply:
Now that you moved it to `docker-compose.ci.override.yml`, we could remove the mounts from this file?